src/builder.rs

```diff
@@ -27,6 +27,46 @@ use crate::{
 
 const MIN_ACTIONS: usize = 2;
 
+/// An enumeration of rules for construction of dummy actions that may be applied to Orchard bundle
+/// construction.
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+pub enum PaddingRule {
+    /// A rule that requires that at least the specified number of Orchard actions be constructed,
+    /// irrespective of whether any genuine actions are included.
+    Require(usize),
+    /// A rule that requires that at least the specified number of Orchard actions be constructed,
+    /// iff any genuine actions are included.
+    PadTo(usize),
+    /// A padding rule that specifies that no additional dummy Orchard actions are to be
+    /// constructed.
+    None,
+}
+
+impl PaddingRule {
+    /// The default padding rule, which ensures that the constructed bundle will contain at least 2
+    /// actions if any genuine Orchard spends or outputs are requested.
+    pub const DEFAULT: PaddingRule = PaddingRule::PadTo(MIN_ACTIONS);
```

Comment:
Nit: the old behaviour is equivalent to `PaddingRule::Require(MIN_ACTIONS)` (which is why we could always return a bundle).